packages/transformer/src/DetachedExportElementAspectsStrategy.ts

```diff
@@ -84,7 +84,7 @@ export class DetachedExportElementAspectsStrategy extends ExportElementAspectsSt
       if(this.excludedElementAspectClassFullNames.has(className))
         continue;
 
-      const getAspectPropsSql = `SELECT * FROM ${className} WHERE ECClassId = :classId ORDER BY Element.Id`;
+      const getAspectPropsSql = `SELECT * FROM ${className.replace(/:(.+)/, ":[$1]")} WHERE ECClassId = :classId ORDER BY Element.Id`;
```

Comment:
how about instead you use `ec_className` in the above query twice to get the className and schemaName separately, and then without a string replace just do:
```ts
`SELECT * FROM [${schemaName}].[${className}] WHERE ...`
```
Since you can have the exact same issue with a poorly named schema

Reply:
also might as well add that case to the test

Reply:
Updated it. It should be a much rarer case, but good catch that we should also handle that.